extensions/sql-database-projects/package.json

```diff
@@ -143,6 +143,11 @@
         "title": "%sqlDatabaseProjects.publish%",
         "category": "%sqlDatabaseProjects.displayName%"
       },
+      {
+        "command": "sqlDatabaseProjects.publishPreview",
```

Comment:
Maybe call this "publishDialog" so that it's more clear; "publishPreview" makes it sound like I'm previewing what I'm publishing.